docker-storage-setup.sh

```diff
@@ -823,6 +830,13 @@ setup_storage() {
   fi
 }
 
+reset_storage() {
+    if [ "$STORAGE_DRIVER" == "devicemapper" ]; then
+	reset_lvm_thin_pool
+    fi
+    rm -f $DOCKER_STORAGE
```

Comment:
Hmm.., strange. I can't seem to find in spec file where we install it. And what's the purpose of default file. Default file is just supposed to be empty and user is not supposed to touch it anyaway. I see following which installs default /etc/sysconfig/docker-storage-setup.

Something is not adding up.

install -d %{buildroot}%{_sysconfdir}/sysconfig
install -p -m 644 %{repo}-storage-setup-override.conf %{buildroot}%{_sysconfdir}/sysconfig/%{repo}-storage-setup